modules/vaos/app/services/vaos/v1/fhir_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../base_service'
+require 'common/exceptions/external/backend_service_exception'
+
+module VAOS::V1
+  class FHIRService < VAOS::BaseService
+    STATSD_KEY_PREFIX = 'api.vaos.fhir'
+
+    def perform(http_method, path, params = nil)
+      super(http_method, path, params, headers)
+    end
+
+    private
+
+    def config
+      VAOS::V1::FHIRConfiguration.instance
+    end
+
+    def headers
+      super.merge!('Content-Type' => 'application/json+fhir')
```

Comment:
Why `merge!` instead of `merge` -- i know it only gets called once, but it seems more functional in terms of non idempotence.